libbeat/outputs/elasticsearch/client.go

```diff
@@ -290,7 +282,7 @@ func bulkEncodePublishRequest(
 			log.Errorf("Failed to encode event meta data: %+v", err)
 			continue
 		}
-		if opType, err := event.GetMetaStringValue(opTypeKey); err == nil && opType == opTypeDelete {
+		if opType, err := events.GetMetaStringValue(*event, events.FieldMetaOpType); err == nil && opType == events.FieldMetaOpTypeDelete.String() {
```

Comment:
do we still store a string for `opType` in the event? Shouldn't it be the enum. Maybe a helper `events.GetOpType(*event)` would be helpful.